gsrs-spring-legacy-indexer/src/main/java/ix/core/search/text/TextIndexer.java

```diff
@@ -2598,6 +2600,20 @@ public void add(EntityWrapper ew) throws IOException {
         add(ew,!shouldNotAdd);
         
     }
+    
+    private boolean shouldIndexAsIdentifier(EntityInfo ei, String field) {
+
```

Comment:
I would add this note here, or somewhere  
        // Identifiers are fields considered worth matching exactly, as opposed to a general text field.
        // Allows for searches to have an easy way to search identifier-level things (e.g. names, codes, uuids, inchis)